modules/subscriptions/views.php

```diff
@@ -320,6 +320,7 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 				if ( ! isset ( $_GET['subscribe'] ) || 'success' != $_GET['subscribe'] ) { ?>
                     <p id="subscribe-email">
                         <label id="jetpack-subscribe-label"
+                               class="screen-reader-text"
```

Comment:
While most modern themes now ship with this class defined (as [per this](https://make.wordpress.org/themes/2015/01/26/supporting-screen-reader-text/)), some old themes still don't. I would consequently recommend adding the following to `modules/subscriptions/subscriptions.css`:

```css
/*
Text meant only for screen readers.
Provides support for themes that do not bundle this CSS yet.
@see https://make.wordpress.org/accessibility/2015/02/09/hiding-text-for-screen-readers-with-wordpress-core/
***********************************/
.screen-reader-text {
	border: 0;
	clip: rect(1px, 1px, 1px, 1px);
	clip-path: inset(50%);
	height: 1px;
	margin: -1px;
	overflow: hidden;
	padding: 0;
	position: absolute ! important;
	width: 1px;
	word-wrap: normal ! important;
}
```

Reply:
@jeherve Thanks, I wondered about that. I do see that this style rule is already included in `social-icons.css`:

https://github.com/Automattic/jetpack/blob/48db8b56a1f81cf31a2dd2982d08cbe4497e8f7f/modules/widgets/social-icons/social-icons.css#L59-L75

So that would mean it would be it would be duplicated. Is there any logic when building `jetpack.css` to remove duplicate rules? This wouldn't help in AMP since `jetpack_implode_frontend_css` is turned off in AMP responses, so then it would need to be the responsibility of the AMP plugin to remove the duplicates as part of the tree shaking. 

Alternatively, the `.screen-reader-text` rule could be extracted and added as a common `<style>` element directly in the page.

Just some thoughts that came to mind. I'll go ahead and amend `subscriptions.css` for now.

Reply:
Yeah, we have not really considered deduplicating content in the big `jetpack.css` file yet, but that is certainly something worth considering down the road. 

Until then, adding the rules to each file so they can be used by those who do not use the concatenation is the best we can do, I think.